cql2pgjson/src/test/java/org/folio/cql2pgjson/CompoundIndexTest.java

```diff
@@ -65,6 +68,41 @@ public void multiFieldnamesMultipartFT() throws Exception {
     String expected = "WHERE to_tsvector('simple', concat_space_sql(tabled.jsonb->'proxy'->'personal'->>'city' , tabled.jsonb->'proxy'->'personal'->>'state')) @@ replace((to_tsquery('simple', ('''Boston''')) && to_tsquery('simple', ('''MA''')))::text, '&', '<->')::tsquery";
     assertEquals(expected, sql);
   }
+
+  @Test
+  public void multiFieldnamesFTStar() throws Exception {
+    CQL2PgJSON cql2pgJson = new CQL2PgJSON("tablea");
+    cql2pgJson.setDbSchemaPath("templates/db_scripts/compoundIndex.json");
+    String sql = cql2pgJson.toSql("ftfieldstar = \"Boston MA\"").toString();
+    String expected = "WHERE to_tsvector('simple', concat_space_sql(concat_array_object_values(tablea.jsonb->>'field1','city') , concat_array_object_values(tablea.jsonb->>'field2','state'))) @@ replace((to_tsquery('simple', ('''Boston''')) && to_tsquery('simple', ('''MA''')))::text, '&', '<->')::tsquery";
+    assertEquals(expected, sql);
+  }
+
+  @Test
+  public void multiFieldnamesFTDotStar() throws Exception {
+    CQL2PgJSON cql2pgJson = new CQL2PgJSON("tablea");
+    cql2pgJson.setDbSchemaPath("templates/db_scripts/compoundIndex.json");
+    String sql = cql2pgJson.toSql("ftfielddotstar = \"Boston MA\"").toString();
+    String expected = "WHERE to_tsvector('simple', concat_space_sql(concat_array_object_values(tablea.jsonb->'field3'->>'info','city') , concat_array_object_values(tablea.jsonb->'field3'->>'info','state'))) @@ replace((to_tsquery('simple', ('''Boston''')) && to_tsquery('simple', ('''MA''')))::text, '&', '<->')::tsquery";
+    assertEquals(expected, sql);
+  }
+
+  @Test
+  public void multiFieldnamesGINStar() throws Exception {
+    CQL2PgJSON cql2pgJson = new CQL2PgJSON("tablea");
+    cql2pgJson.setDbSchemaPath("templates/db_scripts/compoundIndex.json");
+    String sql = cql2pgJson.toSql("ginfieldstar == \"Boston MA\"").toString();
+    String expected = "WHERE lower(concat_space_sql(concat_array_object_values(tablea.jsonb->>'field1','city') , concat_array_object_values(tablea.jsonb->>'field2','state'))) LIKE lower('Boston MA')";
+    assertEquals(expected, sql);
+  }
+  @Test
+  public void multiFieldnamesGINDotStar() throws Exception {
+    CQL2PgJSON cql2pgJson = new CQL2PgJSON("tablea");
+    cql2pgJson.setDbSchemaPath("templates/db_scripts/compoundIndex.json");
+    String sql = cql2pgJson.toSql("ginfielddotstar == \"Boston MA\"").toString();
+    String expected = "WHERE lower(concat_space_sql(concat_array_object_values(tablea.jsonb->'field3'->>'info','city') , concat_array_object_values(tablea.jsonb->'field3'->>'info','state'))) LIKE lower('Boston MA')";
+    assertEquals(expected, sql);
+  }
```

Comment:
Can you add support for string arrays as described in the `languages[*]` example in RMB-474?